administrator/templates/atum/scss/pages/_com_content.scss

```diff
@@ -27,4 +27,9 @@
 			}
 		}
 	}
+
+	section.options-fieldset {
```

Comment:
We really should be doing the fieldset legend in all the sections so a more generic class will probably be required

Reply:
This is adding a white background on the fieldset. The class isn't to do with the legend